### ISSUE TITLE
Access to Koa context in controller

### ISSUE THREAD

**Issue by Gyunikuchan** [lukeautry/tsoa#175]
Would be great if the users of the Koa router can still have access to the context object from within the controller.

```
@SuccessResponse(200, "Found")
@Get("v1/")
public async list(): Promise<MyClass[]> {
	const list = await MyClass.list();
	this.context.SOMEFLAG = true;
	return list;
}
```

**Comment by dgreene1** [comment]
@Gyunikuchan I'm working on a fix for this under issue #388. We'll track the progress there.

**Comment by dgreene1** [comment]
@Gyunikuchan actually, no code fix was necessary. Here's an example of how to get the Koa `ctx` object: https://github.com/lukeautry/tsoa/pull/392/files#diff-26205fa396afae7e698346556c23f256R253
